plugins/reach/lib/kReachUtils.php

```diff
@@ -177,4 +214,81 @@ public static function reachStrToTime($offset , $value)
 		return $result;
 	}
 
+	/**
+	 * @return string
+	 */
+	public static function getVendorCatalogItemsCsvHeaders()
+	{
+		$headers = "ID,Status,vendorPatnerID,Name,systemName,serviceFeature,serviceType,Turn Around Time,Source Language,Target Language,Created at,Updated at,Enable speaker ID,Fixed Price Addons,Price Per Unit,Price Function\n";
+		return $headers;
+	}
+
+	/**
+	 * @param $csvData
+	 * @return array
+	 */
+	public static function validateAndTranslateCatalogItemCsvData($csvData)
+	{
+		if (isset($csvData['createdAt']))
```

Comment:
Also here - you can hold an array of string with the names of the fields you want, then just foreach them